app/src/main/java/com/neo/speaktouch/model/Type.kt

```diff
@@ -30,76 +29,104 @@ import android.widget.RadioButton
             if (className `is` ImageView::class.java) {
-                return if (node.isClickable) BUTTON else IMAGE
+                return if (node.isClickable) Button else Image
             }
 
             /* TextView */
 
-            // View->TextView->EditText
-            if (className `is` EditText::class.java) return EDITFIELD
+            // View -> TextView -> EditText
+            if (className `is` EditText::class.java) return EditField
 
-            // View->TextView->CheckedTextView
-            if (className `is` CheckedTextView::class.java) return CHECKEDTEXT
+            // View -> TextView -> CheckedTextView
+            if (className `is` CheckedTextView::class.java) return Checkable.TextView
 
             /* Button */
 
-            // View->TextView->Button->CompoundButton->Switch
-            if (className `is` Switch::class.java) return SWITCH
+            // View -> TextView -> Button -> CompoundButton -> Switch
+            if (className `is` Switch::class.java) return Checkable.Switch
 
-            // View->TextView->Button->CompoundButton->ToggleButton
-            if (className `is` ToggleButton::class.java) return TOGGLE
+            // View -> TextView -> Button -> CompoundButton -> ToggleButton
+            if (className `is` ToggleButton::class.java) return Checkable.Toggle
 
-            // View->TextView->Button->CompoundButton->RadioButton
-            if (className `is` RadioButton::class.java) return RADIO
+            // View -> TextView -> Button -> CompoundButton -> RadioButton
+            if (className `is` RadioButton::class.java) return Checkable.Radio
 
-            // View->TextView->Button->CompoundButton->CheckBox
-            if (className `is` CheckBox::class.java) return CHECKBOX
+            // View -> TextView -> Button -> CompoundButton -> CheckBox
+            if (className `is` CheckBox::class.java) return Checkable.Checkbox
 
-            // View->TextView->Button
-            if (className `is` Button::class.java) return BUTTON
+            // View -> TextView -> Button
+            if (className `is` android.widget.Button::class.java) return Button
 
             /* AdapterView */
 
-            // View->ViewGroup->AdapterView->AbsListView
-            if (className `is` AbsListView::class.java) return LIST
+            // View -> ViewGroup -> AdapterView -> AbsListView
+            if (className `is` AbsListView::class.java) return List
 
-            // View->ViewGroup->AdapterView->AbsSpinner
-            if (className `is` AbsSpinner::class.java) return OPTIONS
+            // View -> ViewGroup -> AdapterView -> AbsSpinner
+            if (className `is` AbsSpinner::class.java) return DropdownList
 
             /* Independent of inheritance */
 
-            if (node.collectionInfo != null) return LIST
+            if (node.collectionInfo != null) return List
 
-            if (node.isHeading) return TITLE
+            if (node.isHeading) return Title
```

Comment:
Testar combinação de titulo e outros tipos em outros leitores.

Reply:
To my surprise, Talkback read the isHeading property in combination with other types. What do you think about this, @PatrykMis? To me, it doesn't make much sense for an image, button, or switch to be a title

Reply:
You can mark it as a title with `android:accessibilityHeading="true"` in the xml or with `ViewCompat.setAccessibilityHeading`.

Reply:
Strangely, Jieshou and Prudence did not recognize the signaling.

Reply:
Definitely "heading" is more appropriate than "title". "Title" means rather title of a window, while "title" in this context means heading of a section.

Reply:
Are you suggesting changing "title" to "heading"? We can do this. But what about the issue of combinations?

Reply:
For now, we will continue with the current behavior, which is different from that of TalkBack. I have added a comment for documentation purposes.